### PR TITLE
First revision on common use-cases.

### PR DESCRIPTION
**Pull request by sabbyanandan**

### CODE REVIEW

File: spring-xd-use-cases.tex [hunk 1]
```diff
@@ -1,2 +1,11 @@
...
+\item \textbf{24/7 Production Pipelines}: The business demands highly available data streams with guaranteed data processing, to react in real-time accurately. Spring XD's runtime is highly reliable that can recover from failures seamlessly. Production running pipelines are long running tasks - there's no end. For ad-hoc operations such as querying, machine learning, or data crunching, `taps' in Spring XD are commonly adopted to fork the data from primary pipeline. This results in no disruption with the primary pipeline, at the same time ad-hoc demands can be fulfilled. 
+\item \textbf{Closed-loop Analytics}: Spring XD orchestrates the entire analytics loop - gathering data from any source, triggering actions, handling feedback loops from machine learning models, and computing real-time predictions.
+\item \textbf{Hadoop and Beyond}: Enterprises with existing investment in Hadoop technologies find Spring XD as one-stop orchestration platform. Whether it is MapReduce, Hive, Pig, or HBase scripts - developer experience is the same.
```
Comment (cppwfs) [inline spring-xd-use-cases.tex:10]: `as **the** one-stop orchestration platform.`  Add "the" to this line.

Comment (cppwfs) [inline spring-xd-use-cases.tex:10]: `Spring XD, as **a** one-stop runtime,`

Comment (cppwfs) [inline spring-xd-use-cases.tex:10]: 'Spring XD, as one-stop runtime, provides data integration adapters to consume data of varied formats (i.e., structures, unstructured, binary, ..) that reside in different toolchains (i.e., database, middlewares, in-memory grids, ..). '   This statement left me hanging.  It consumes data and then what?

**Comment by pperalta** [comment]
Merged as 07e81e544171ef6535d77a9bfea3577c2f0ebb9c

**Pull request closed without merging**